app/v5/bfLosses/list/def1/model/response/Def1_ListBFLossesItem.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v5.bfLosses.list.def1.model.response
+
+import api.models.domain.TaxYear
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{Json, OWrites, Reads, __}
+import v5.bfLosses.list.model.response.ListBFLossesItem
+import v5.bfLosses.list.model.{IncomeSourceType, LossType, TypeOfLoss}
+
+case class Def1_ListBFLossesItem(lossId: String,
```

Comment:
If `ListBFLossesItem` is removed, this can be renamed `ListBFLossesItem`.

Reply:
I assume you mean if the trait is removed, the "Def1" bit can be removed from this code? Please could you clarify?

Reply:
Just read the next comment and agree- will implement change.